hs/Scripts.hs

```diff
@@ -5,59 +5,59 @@ module Scripts where
 import           Text.Blaze ((!))
 import qualified Text.Blaze.Html5 as H
 import qualified Text.Blaze.Html5.Attributes as A
-import MakeElements
+import Utilities
 
-plannerScripts :: H.Html
-plannerScripts = concatHtml (map makeScript["https://ajax.googleapis.com/ajax/libs/jquery/1.10.2/jquery.min.js",
-                                           "https://ajax.googleapis.com/ajax/libs/jqueryui/1.10.4/jquery-ui.min.js",
-                                           "static/js/common/modal.js",
-                                           "static/js/graph/objects/edge.js",
-                                           "static/js/graph/objects/node.js",
-                                           "static/js/common/objects/course.js",
-                                           "static/js/common/cookie_handler.js",
-                                           "static/js/common/course_description.js",
-                                           "static/js/graph/sidebar/focuses.js",
-                                           "static/js/graph/sidebar/fce_count.js",
-                                           "static/js/common/objects/section.js",
-                                           "static/js/common/utilities/util.js",
-                                           "static/js/graph/utilities/structs.js",
-                                           "static/js/graph/utilities/util.js",
-                                           "static/js/graph/create_data.js",
-                                           "static/js/graph/parse_graph.js",
-                                           "static/js/graph/mouse_events.js",
-                                           "/static/js/common/image_conversion.js",
-                                           "/static/js/common/graph_image.js",
-                                           "/static/js/common/facebook/facebook_login.js",
-                                           "/static/js/common/facebook/facebook_image.js",
-                                           "/static/js/common/facebook/facebook_modal.js",
-                                           "static/js/graph/setup.js",
-                                           "static/js/post/update_post.js",
-                                           "static/js/graph/sidebar/sidebar_divs.js",
-                                           "static/js/graph/sidebar/sidebar_events.js",
-                                           "static/js/graph/sidebar/focus_descriptions.js",
-                                           "static/js/common/export/export.js"])
+graphScripts :: H.Html
+graphScripts = concatHtml (map makeScript["https://ajax.googleapis.com/ajax/libs/jquery/1.10.2/jquery.min.js",
+                                          "https://ajax.googleapis.com/ajax/libs/jqueryui/1.10.4/jquery-ui.min.js",
+                                          "/static/js/common/modal.js",
```

Comment:
I changed the indentation here already in the `facebook-config` branch, which was recently merged in. You should probably do a merge from master to grab that.